Source/SwiftLintFramework/Rules/Style/SortedImportsRule.swift

```diff
@@ -47,96 +53,25 @@ private extension Sequence where Element == Line {
 }
 
 struct SortedImportsRule: CorrectableRule, ConfigurationProviderRule, OptInRule {
-    var configuration = SeverityConfiguration(.warning)
+    var configuration = SortedImportsConfiguration(grouping: .name)
 
+    private static let examples = SortedImportsRuleExamples.self
```

Comment:
Please inline this variable, i.e. use `SortedImportsRuleExamples.nonTriggeringExamples`, ...